helm/charts/hpe-csi-driver/templates/nimble-csp.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.nimble.enabled }}
```

Comment:
Reflect desired changes.

Reply:
What you mean with "reflect" what shall I do here?

Reply:
Somehow my first review comment was lost. We need the parameters changed and verbiage updated. Those changes needs to be reflected in the respective template.

|Param|Desc|Default|
| - | - | -|
| disable.nimble | Do not apply Nimble CSP `Service` and `Deployment` | false |
| disable.primera | Do not apply Primera CSP `Service` and `Deployment` | false |

Reversing enable to disable is simply to be in line with `disableNodeConformance` where you need to flag something "true" to change a non-default behaviour.